src/bio2bel/sources/intact.py

```diff
@@ -0,0 +1,404 @@
+
+    :param graph: graph to add edges to
+    :param row: row with metainformation about source, target, relation, pubmed_id
+    :return: None
+    """
+    relation = row[RELATION]
+    source_uniprot_id = row[SOURCE]
+    target_uniprot_id = row[TARGET]
+
+    pubmed_ids = row['pubmed_ids']
+    # pubmed_ids = pubmed_ids.split('|')
+
+    source = pybel.dsl.Protein(
+        namespace='uniprot',
+        identifier=source_uniprot_id,
+        name=get_mnemonic(source_uniprot_id),
+    )
+    target = pybel.dsl.Protein(
+        namespace='uniprot',
+        identifier=target_uniprot_id,
+        name=get_mnemonic(target_uniprot_id),
+    )
+
+    for pubmed_id in pubmed_ids:
+
+        # INCREASE
+        if relation in INTACT_INCREASES_ACTIONS:
+
+            if relation == 'phosphorylation reaction':
+                target_ub = target.with_variants(
+                    pybel.dsl.ProteinModification('Ph')
+                )
+            elif relation == 'sumoylation reaction':
+                target_ub = target.with_variants(
+                    pybel.dsl.ProteinModification('Sumo')
+                )
+            elif relation == 'methylation reaction':
+                target_ub = target.with_variants(
+                    pybel.dsl.ProteinModification('Me')
+                )
+            elif relation == 'transglutamination reaction':
+                target_ub = target.with_variants(
+                    pybel.dsl.ProteinModification('Me')
+                )
+            elif relation == 'ubiquitination reaction':
+                target_ub = target.with_variants(
+                    pybel.dsl.ProteinModification('Ub')
+                )
+            elif relation == 'acetylation reaction':
+                target_ub = target.with_variants(
+                    pybel.dsl.ProteinModification('Ac')
+                )
+            elif relation == 'adp ribosylation reaction':
+                target_ub = target.with_variants(
+                    pybel.dsl.ProteinModification('ADPRib')
+                )
+            elif relation == 'neddylation reaction':
+                target_ub = target.with_variants(
+                    pybel.dsl.ProteinModification('Nedd')
+                )
+            elif relation == 'hydroxylation reaction':
+                target_ub = target.with_variants(
+                    pybel.dsl.ProteinModification('Hy')
+                )
+            elif relation == 'phosphotransfer reaction':
+                target_ub = target.with_variants(
+                    pybel.dsl.ProteinModification('Ph')
+                )
+            elif relation == 'glycosylation reaction':
+                target_ub = target.with_variants(
+                    pybel.dsl.ProteinModification('Glyco')
+                )
+            elif relation == 'palmitoylation reaction':
+                target_ub = target.with_variants(
+                    pybel.dsl.ProteinModification('Palm')
+                )
```

Comment:
needs an else. if you never think that else should happen, then else with `raise ValueError`

Reply:
Fixed here: 2f971522f3448bd2f187d2d9b97e1ccf3f913c5d